docker-compose.yml

```diff
@@ -167,6 +167,13 @@ services:
         tag: "desec-ns/prometheus"
     restart: unless-stopped
 
+  kdig:
+    image: cznic/knot:master
+    networks:
+      front:
+        ipv4_address: 10.16.0.4
+    restart: 'no'
```

Comment:
This is probably fired up during `docker-compose up -d`, and then exists quickly, right? Can autostart be avoided?

Reply:
It seems [there is a way](https://stackoverflow.com/questions/37254881/is-there-any-way-to-disable-a-service-in-docker-compose-yml) but it requires docker compose 1.28 and docker-compose.yml version 3.9. Not sure if it is worth upgrading for this.

Reply:
Fair, that seems not necessary right now.

Thinking about alternative solutions, it doesn't seem to me that a service is needed at all. One can just have a `kdig.sh` script that runs `docker run --network desec-ns_front cznic/knot:master kdig @10.16.0.2 "$@"`, and then issue queries as `./kdig.sh A desec.io`.

How about we put a script like that in the repo? (The IP address above is for dnsdist; you'd probably want to set it to 10.16.0.3 for the DoQ proxy.)

Reply:
I like the idea, but unfortunately the network name varies. Sometimes it's `desecns_front`, sometimes `desec-ns_front`.

The script could check the network name, but `docker network ls` takes a while.

Reply:
I take that back, `docker network ls` is not very slow. I've created a script as you suggested and swapped out the second commit in this PR. Note that the script can only be used for DoQ queries. Try:

    ./kdig-doq.sh _signal.$DESEC_NS_NAME SOA

with the $DESEC_NS_NAME from your `.env` file.

Reply:
Great!